src/codegen/codegen_ispc_visitor.cpp

```diff
@@ -63,6 +63,39 @@ void CodegenIspcVisitor::visit_var_name(ast::VarName& node) {
     CodegenCVisitor::visit_var_name(node);
 }
 
+void CodegenIspcVisitor::visit_local_list_statement(ast::LocalListStatement& node) {
+    if (!codegen) {
+        return;
+    }
+    auto type = CodegenCVisitor::local_var_type() + " ";
+    printer->add_text(type);
+    auto local_variables = node.get_variables();
+    std::set<std::string> uniform_variables;
```

Comment:
`uniform_variables` variable seems unused

Reply:
True. I missed it 😞